docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -40,6 +50,9 @@ endif::[]
 ifdef::mta[]
 `https://marketplace.eclipse.org/content/migration-toolkit-applications-mta` and click *OK*.
 endif::[]
+. Select the Java version to use.
+.. If using Java 17, leave the *JRE* selection unchanged.
+.. If using Java 11, in *JRE* select *JavaSE 11*.
```

Comment:
@sbeskin-redhat should this be
```suggestion
.. If using Java 11, in *JRE*, select *JavaSE 11*.
```
It is an awkward sentence

Reply:
@anarnold97 
I suggested a change above.